splunk/dasg_metrics_dashboard.xml

```diff
@@ -381,12 +411,55 @@
       <title>4. TOTAL V2 FHIR Calls Made (w/ successful status)</title>
       <single>
         <search>
-          <query>| makeresults | eval total=$tokFhirV1OKCount$ + $tokFhirV2OKCount$ | table total</query> 
+          <query>| makeresults | eval total=$tokFhirV1OKCount$ + $tokFhirV2OKCount$ | table total</query>
         </search>
         <option name="drilldown">all</option>
         <option name="height">226</option>
         <option name="refresh.display">progressbar</option>
       </single>
     </panel>
   </row>
-</form>
+  
+  <!--SANDBOX Metrics start-->
+  
+  <row>
+    <panel>
+      <html>
+       <u1> <h1>
+            <b>SBX DASG Metrics Logging Event Counts for Group Timestamp - $tokGroupTimestamp$</b>
+          </h1>
+        </u1>
+     </html>
+    </panel>
+  </row>
+  <row>
+    <panel>
+      <title>1. TOTAL Dev w/ SBX Cred + Registered App</title>
+      <single>
+        <search>
```

Comment:
The `message.group_timestamp` value is set when the logging is run via the Django command to tie events together. It will be different for runs in each ENV (impl, prod).  You should have a different `tokGroupTimestampSandbox` token to be used in this section.

Similar to 1c. I noticed that the counts don't update in this section when changing the date range.


```suggestion
      <title>1. TOTAL Dev w/ SBX Cred + Registered App</title>
        <search>
            <init>
              <set token="tokGroupTimestampSandbox">$result.message.group_timestamp$</set>
            </init>
            <query>index=bluebutton source="/var/log/pyapps/perf_mon.log*" host=* env=impl | search "message.type"="global_state_metrics" | head 1 | table message.group_timestamp</query>
        </search>  
      <single>
        <search>
```

Reply:
@dtisza1 - Also good feedback - I misunderstood the need here as well, what you are pointing out makes good sense.